pkg/workload/workload.go

```diff
@@ -540,6 +540,15 @@ func (o Ordering) GetQueueOrderTimestamp(w *kueue.Workload) *metav1.Time {
 			return &evictedCond.LastTransitionTime
 		}
 	}
+	if !features.Enabled(features.PrioritySortingWithinCohort) {
+		if preemptedCond := apimeta.FindStatusCondition(w.Status.Conditions, kueue.WorkloadPreempted); preemptedCond != nil &&
+			preemptedCond.Status == metav1.ConditionTrue &&
+			preemptedCond.Reason == kueue.InCohortReclaimWhileBorrowingReason {
```

Comment:
We would need to do `strings.HasPrefix(preemptedCond.Reason, "InCohort")` if we want to cherry-pick down to 0.7. 
cc @gabesaba

Reply:
As far as I can tell we don't need to cherry-pick to 0.7. Closing the comment.